packages/buffetjs-styles/src/components/InputNumber/index.js

```diff
@@ -9,7 +9,10 @@ import colors from '../../assets/styles/colors';
 import sizes from '../../assets/styles/sizes';
 import img from '../../assets/icons/icon_select.svg';
 
+// TODO
```

Comment:
Do you still need this comment ?

Reply:
Yes this one is needed we need to update the file so we can pass the stylelint but it needs to be done in another PR.